app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -841,6 +841,11 @@ export class Dispatcher {
     return this.appStore._setConfirmRepoRemoval(value)
   }
 
+  public revealInFileManager(repository: Repository, filepath: string): boolean {
```

Comment:
Can we document this please, specifically with information about how the path parameter is expected to be a path relative to the repository root. Could we also rename 'filepath' to either just 'path' or (preferably) 'relativeFilePath'